internal/rollup/rollup.config.js

```diff
@@ -0,0 +1,59 @@
+const rollup = require('rollup');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const fs = require('fs');
+const path = require('path');
+
+class NormalizePaths {
+  resolveId(importee, importer) {
+    // process.cwd() is the execroot and ends up looking something like /.../2c2a834fcea131eff2d962ffe20e1c87/bazel-sandbox/872535243457386053/execroot/<workspace_name>
+    // from the path to the es6 output is bazel-out/host/bin/<build_file_path>/rollup.runfiles/<workspace_name>/<build_file_path>/bazel-out/host/bin
+    const workspaceName = process.cwd().split('/').pop();
+
+    const firstSegment = importee.split('/')[0]
+    const importerDir = importer ? path.dirname(importer) : ""
+    const buildFileMatch = importerDir.match(`rollup\\.runfiles\\/${workspaceName}/([\\w\\-. \\/]+)\\/bazel\\-out`)
+    const buildFilePath = buildFileMatch ? buildFileMatch[1] : "";
+
+    if (firstSegment === 'bazel-out') {
+      // entry point is a relative path from the execroot
+      return `${process.cwd()}/${importee}.js`;
+    } else if (firstSegment === '.' || firstSegment === '..') {
+      // relative import
+      return `${importerDir}/${importee}.js`;
+    } else if (firstSegment === workspaceName || (buildFilePath && firstSegment === buildFilePath)) {
+      // most imports start with the workspace name but some start with the build file path
```

Comment:
This is where things get overly complicated. I derived both the build file path from the `importer` and the workspace name from the execroot (`process.cwd()`). Then I use this convert imports such as `angular_bazel_example/src/app` to an absolute path from the execroot (`process.cwd()`). There is probably a better way to do this.

Reply:
Is the idea here to provide an internal, fixed rollup-config?
I was thinking we would need to give the consumer the freedom to provide their own config that we would accept as a dependency. 

In practice I think it will be difficult to provide a "catch all" version since different apps will have different plugin needs etc.

Reply:
That said, if we are mainly targeting Angular with this, it's probably possible to come up with a single config that will work for most projects.

Reply:
We can't target only Angular projects with this - anything in rules_nodejs should work for the general case, like a React app written in JavaScript.

In general, our principle should be to expose configuration ability to users only as we understand the use cases for it. In some cases, exposing the configuration file from the framework/library/language itself is okay (we do this with `tsconfig.json`) but in other cases we might want to expose only some attributes on the rule.

The reasoning is that it's easy to give the user more options, but very hard to take any options away. I'd rather give Bazel users simplicity and composability (rules tend to work with each other) in preference to freedom.

Also, most configuration files tell the tool about your program, but Bazel already knows about your program, so I think a lot of the draw to Bazel should essentially be "no more config files".

Reply:
Yes. Whatever options that are needed can be exposed as rule attributes. I can see a need for optionally turning off uglify in the rule.